www/install/php/Update-22.04.0-beta.2.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
move this script in `www/install/php/Update-22.04.0-beta.1.php`

Reply:
There is already a script named Update-22.04.0-beta.1.php in the branch develop. Should I merge these files ?

Reply:
Yes